src/i18n/events.ts

```diff
@@ -0,0 +1,277 @@
+export type Category = 'workshop' | 'event'
+export type TimeStatus = 'passed' | 'thisWeek' | 'coming' | undefined
+export type CtaIdType = 'cfp' | 'donation' | 'sign-up' | 'video' | 'website'
+export type City = string // Create special type based of messages (keyof typeof as const)
+
+interface Sponsor {
+  name: string
+  link: string
+  image: {
+    url: string
+    alt: string
+  }
+}
+
+interface CtaType {
+  id: CtaIdType
+  href: string | null
+}
+
+interface Detail {
+  id: string
+  icon: string
+  text: string
+}
+
+export interface Event {
+  arguments: string
+  category: Category
+  community_sponsors?: Sponsor[]
+  ctas?: CtaType[]
+  description: string
+  id: number
+  shortDescription: string
+  speakers: string
+  sponsors?: Sponsor[]
+  subtitle?: string
+  ticketsURL?: string
+  title: string
+  details: Detail[]
+  date: {
+    day: string
+    start: string
+    end: string
+  }
+  image: {
+    url: string
+    alt: string
+  }
+  location: {
+    name: string
+    city: string
+    url: string
+  }
+}
+
+export const events: Event[] = [
```

Comment:
I feel that we could have a JSON file in order to store our events OR, maybe better, a folder containing different JSON files: one json file will be an event.

Would it be possible to handle like this?

Reply:
Totally, it's going to be even more easier to handle it like that. We could have 'em all in separate JSON files and then export them from a single TypeScript file 👍